getting-started/setup-multi-node-basic.md

```diff
@@ -23,30 +24,70 @@ node, and follow the instructions in the next section.  More information will be
 forthcoming as TimescaleDB multi-node is made available on these cloud platforms.
 
 ## Initialize data nodes from the access node [](init_data_nodes_on_access_node)
-While connected to the access node (psql), use the command:
+
+Once logged in on the access node, it is necessary to add data nodes
+to the local database before creating distributed hypertables. This
+will make the data node available for use by distributed hypertables
+and the access node will also connect to the data node and initialize
+it.
+
+While connected to the access node as a superuser (e.g., via `psql`),
+use the command:
 
 ```sql
 SELECT add_data_node('example_node_name', host => 'example_host_address');
 ```
 
-`example_node_name` should be a unique name for the node. `example_host_address` 
-is the host name or IP address of the data node.
+`example_node_name` should be a unique name for the
+node. `example_host_address` is the host name or IP address of the
+data node. It is also possible to specify a password to authenticate
+with using the optional `password` parameter. But this is only
+necessary if password authentication is used to connect to data nodes
+and the password is not provided through other means (e.g., a local
+password file). See the [`add_data_node`][add_data_node] API reference
+documentation for detailed information about this command.
+
+It should now be possible to create distributed hypertables. Note,
+however, that in order to create and use distributed hypertables as a
+non-superuser, the user role needs to be distributed as well (i.e.,
+exist on all data nodes) and requires privileges to use the data nodes
```

Comment:
Otherwise, it sounds more like you always need ot take this step, which you often might not need to do for testing.
```suggestion
It should now be possible to create distributed hypertables, e.g., 
running as the default `postgres` superuser.  If you wish to
create and use distributed hypertables as a
non-superuser, the user role needs to be distributed as well (i.e.,
```